gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
```

Comment:
If gradle version upgrade is necessary, please move that to a separate PR as well.

Reply:
> Please limit scope of the PR just to migration from maven to gradle build system. Anything not directly related to maven->gradle migration such as UT, java doc, JDK version upgrade should be part of a separate PR(s).

I've excluded java doc from compilatoin, I can remove the change in java doc. 
But all other changes are necessary. If we move all these changes to separate PRs, we'd have to create quite a few PRs and merge them before this PR. 
Would that help if I modify the title of the PR and add some description for those changes in the PR description to clarify the scope of the PR?